terraform/environments/core-network-services/firewall.tf

```diff
@@ -228,11 +228,35 @@ resource "aws_networkfirewall_rule_group" "global_protect" {
             }
           }
         }
+        stateless_rule { # Azure NOMIS test to MP Nomis database
+          priority = 200
+          rule_definition {
+            actions = ["aws:pass"]
+            match_attributes {
+              source {
+                address_definition = "10.101.0.0/16" # Azure NOMIS Test
+              }
+              source_port {
+                from_port = 1024
```

Comment:
If I understand correctly, this configuration allows any unprivileged source port in the range 1024-65535. Just wondering if this might be too open or if there's any reference on why we need it it be so open. Is this to allow NAT-ports?